### PR TITLE
Necessary refactor for protobuf v4

### PR DESCRIPTION
**Pull request by snazy**
* Bump submodules cel-spec + googleapis
* Move symlinks in `cel-generated-pb` around
* Use `testFixtures` instead of "test-jar" (less boilerplate)
* Remove jandex plugin (conflicts with the protobuf plugin all the time)

### CODE REVIEW

File: core/src/testFixtures/java/org/projectnessie/cel/Util.java [hunk 1]
```diff
@@ -15,13 +15,11 @@
  */
 package org.projectnessie.cel;
 
-import static org.assertj.core.api.Assertions.assertThat;
```
Comment (adutra) [inline core/src/testFixtures/java/org/projectnessie/cel/Util.java:18]: Why this change?

Comment (snazy) [reply core/src/testFixtures/java/org/projectnessie/cel/Util.java:18]: Hm - i only moved the file actually

**Review by adutra — APPROVED** [review]
